arch/arm/core/aarch32/cortex_m/mpu/arm_core_mpu.c

```diff
@@ -77,6 +77,14 @@ static const struct z_arm_mpu_partition static_regions[] = {
 		.attr = K_MEM_PARTITION_P_RX_U_RX,
 		},
 #endif /* CONFIG_ARCH_HAS_RAMFUNC_SUPPORT */
+#if DT_NODE_HAS_STATUS(DT_CHOSEN(zephyr_itcm), okay)
```

Comment:
@MaureenHelm this will consume one MPU region.
I wonder whether this is needed. Does this memory need to be explicitly accessed by unprivileged code (user mode)?

Reply:
Indeed it does consume an MPU region, which is why I didn't initially set the dts chosen node by default for all i.MX RT boards. I mistakenly thought I needed the MPU region for any execute access. I don't currently see a need for user access, so the MPU region has now been removed.

Reply:
very well